build.gradle

```diff
@@ -20,7 +20,7 @@ ext {
     automatticRestVersion = '1.0.8'
     automatticStoriesVersion = '2.3.0'
     automatticTracksVersion = '3.2.0'
-    gutenbergMobileVersion = 'v1.102.0'
+    gutenbergMobileVersion = '6139-3025c34b5015f43884b697cb0cd80826c97138ce'
```

Comment:
I noticed that this reference is not pointing to the latest commit of the Gutenberg Mobile PR, the SHA should be `273a193d6c0fcb05d09e1722902c8d985f089422`. Nevertheless, since the commits pushed after this reference only introduce an E2E test fix and update the release notes, we could directly use the release Git tag once we merge the Gutenberg PRs.